casper/src/main/scala/coop/rchain/casper/util/comm/BlockApproverProtocol.scala

```diff
@@ -141,7 +142,7 @@ object BlockApproverProtocol {
             .either(())
             .or("Mismatch between number of candidate deploys and expected number of deploys.")
       stateHash <- runtimeManager
-                    .replayComputeState(runtimeManager.emptyStateHash, blockDeploys)
+                    .replayComputeState[Task](runtimeManager.emptyStateHash, blockDeploys)
```

Comment:
Nice! That's much better than what I was doing with the `Sync[Task]` implicit parameter. In fact - how could I've missed that :man_facepalming: :D

Reply:
It was late ;-)